jQuery/honeyHeadquarters.js

```diff
@@ -1,4 +1,27 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
-$.ajax('http://localhost:3000/citizens').done(function(data) {
-  console.log(data);
+$.ajax('http://localhost:3000/citizens').done(function(citizens) {
```

Comment:
Would it damage your soul if, instead of changing everything below to `function() {...}`, I changed this to arrow syntax?